build.gradle.kts

```diff
@@ -19,4 +26,25 @@ dependencies {
     implementation("org.springframework.boot:spring-boot-starter-validation")
     implementation("org.springframework.boot:spring-boot-starter-webflux")
     implementation("io.projectreactor.kotlin:reactor-kotlin-extensions:1.1.2")
+    swaggerCodegen("io.swagger.codegen.v3:swagger-codegen-cli:3.0.25")
+}
+
+swaggerSources {
+    create("storagegrid").apply {
+        setInputFile(file("src/main/resources/swagger/storagegrid-api.yml"))
+        code(
+            closureOf<GenerateSwaggerCode> {
+                language = "kotlin-client"
```

Comment:
Dette skal nok være noe annet for å generere en spring-boot webclient client.

language = "java"
library = "webclient"

Spør om man vil ha clienten i java though. Det bør kanskje testes.

Reply:
Jeg får ikke dette til å funke. Får melding om at `webclient` er et "unknown library". Prøvde også å bytte fra `swaggerCodegen("io.swagger.codegen.v3:swagger-codegen-cli:3.0.25")` til `swaggerCodegen("org.openapitools:openapi-generator-cli:5.0.1")`, som er det som brukes i Storyteller Validator - da fikk jeg ingen feilmelding, men det ble ikke generert noe kode.

Reply:
Har prøvd å debugge litt og jeg tror det kan ha å gjøre med formatet til Yaml-filen. Ser ut til at den er laget for swagger-codegen-cli og ikke openapi-generator-cli. Men swagger-codegen-cli støtter ikke webclient.

Reply:
Fikk det til å funke nå med biblioteket `resttemplate`. Skal vi endre til det?

Reply:
Tror ikke swagger-codegen-cli og openapi-generator-cli i seg selv er problemet. Men openapi funker kun på Swagger 3.0 tror jeg. Ser at aktuell swagger er 2.0. Kanskje bare konvertere den til 3.0? Verdt et forsøk, litt middels å basere seg på 2.0 tipper jeg 🤔 

Jeg kan fikle litt med denne branchen.